install_config/install/advanced_install.adoc

```diff
@@ -784,6 +783,44 @@ openshift_examples_modify_imagestreams=true
 openshift_docker_additional_registries=example.com
 ----
 
+[[advanced-install-configuring-registry-location]]
+=== Configuring the Registry Console
+
+If you are using a registry console image other than the default or require a specific version of the console,
+specify the desired registry within the
+*_/etc/ansible/hosts_* file. 
+
+----
+openshift_cockpit_deployer_prefix=<registry-name>/<namespace>/
+openshift_cockpit_deployer_basename=<my-console>
+openshift_cockpit_deployer_version=<cockpit-image-tag>
+----
+
+.Registry Variables
+[options="header"]
+|===
+
+|Variable |Purpose
+|`*openshift_cockpit_deployer_prefix*`
+|Specify the URL and path to the directory where the image is located.
+
+|`*openshift_cockpit_deployer_basename*`
+|Specify the name of the image.
+
+
+|`*openshift_cockpit_deployer_version*`
+|Specify the Cockpit image verion
+|===
+
+For example:
+If your image is at *_registry.example.com/openshift3/registry-console_* and you require
+version 1.4.1, enter:
+----
+openshift_cockpit_deployer_prefix='registry.example.com/openshift3/' 
+openshift_cockpit_deployer_basename=`registry-console`
```

Comment:
"registry-console" is the default value, in this example, it is better to change it to other than the default name